src/tss2-tcti/tctildr-dl.c

```diff
@@ -32,25 +35,90 @@ struct {
         .description = "Access libtss2-tcti-default.so",
     },
     {
-        .file = "libtss2-tcti-tabrmd.so",
+        .file = "libtss2-tcti-tabrmd.so.0",
         .description = "Access libtss2-tcti-tabrmd.so",
     },
     {
-        .file = "libtss2-tcti-device.so",
+        .file = "libtss2-tcti-device.so.0",
```

Comment:
I'm sure there's a reason, but why prepend the `.0`?

Reply:
I tried to covey this in the commit message but may not have been as clear as possible

> When dynamically loading TCTIs the ESYS default selection and search
> logic tries known TCTI library names with the '.so' suffix. This
> works on systems where the library has been installed from source or
> when development packages are installed. Packages for the TCTI (non
> 'dev' packages) however will provide a library with the library version
> in the suffix (eg '.so.0') causing the existing approach to fail for
> the common case. This commit replaces '.so' suffix with '.so.0' as a
> fix. Both the tools and the tabrmd worked through these issues some
> time ago with help from some of the distro packagers (see 2296d48a from
> tabrmd).

Devs from Suse explained this to me here: https://github.com/tpm2-software/tpm2-abrmd/issues/486

Reply:
RedHat dev patched this in the tabrmd for me: https://github.com/tpm2-software/tpm2-abrmd/pull/489
There are similar patches in the tools as well.

Reply:
Ah thank you. I failed to read the commit messages

Reply:
SHAME! :stuck_out_tongue_closed_eyes: